code/run_iLasso_row.m

```diff
@@ -1,55 +1,27 @@
-function [ALasso, metric] = run_iLasso_row(X,params,rownum)
+function [metric] = run_iLasso_row(m,outs,params,rownum)
```

Comment:
Using the `mat` file to track intermediate state instead of `X`.  `outs` is the array of output file pointers.